### ISSUE TITLE
Change in JPX Market Closing Time

### ISSUE THREAD

**Issue by HinnyTsang** [rsheftel/pandas_market_calendars#350]
### Summary
The closing market time for JPX has been extended to 15:30 PM (local time). The current version doesn't resolve for the change.

Test code:
```python
# test.py
import pandas_market_calendars as mcal

calendar = mcal.get_calendar("XJPX")

schedule = calendar.schedule(
    start_date="2024-10-25", end_date="2024-11-07", tz="Asia/Tokyo"
)
print(f"version: {mcal.__version__}")
print(schedule)
```

Output:
```console
~$python ./test.py
version: 4.4.1
                         market_open               break_start                 break_end              market_close
2024-10-25 2024-10-25 09:00:00+09:00 2024-10-25 11:30:00+09:00 2024-10-25 12:30:00+09:00 2024-10-25 15:00:00+09:00
2024-10-28 2024-10-28 09:00:00+09:00 2024-10-28 11:30:00+09:00 2024-10-28 12:30:00+09:00 2024-10-28 15:00:00+09:00
2024-10-29 2024-10-29 09:00:00+09:00 2024-10-29 11:30:00+09:00 2024-10-29 12:30:00+09:00 2024-10-29 15:00:00+09:00
2024-10-30 2024-10-30 09:00:00+09:00 2024-10-30 11:30:00+09:00 2024-10-30 12:30:00+09:00 2024-10-30 15:00:00+09:00
2024-10-31 2024-10-31 09:00:00+09:00 2024-10-31 11:30:00+09:00 2024-10-31 12:30:00+09:00 2024-10-31 15:00:00+09:00
2024-11-01 2024-11-01 09:00:00+09:00 2024-11-01 11:30:00+09:00 2024-11-01 12:30:00+09:00 2024-11-01 15:00:00+09:00
2024-11-05 2024-11-05 09:00:00+09:00 2024-11-05 11:30:00+09:00 2024-11-05 12:30:00+09:00 2024-11-05 15:00:00+09:00
2024-11-06 2024-11-06 09:00:00+09:00 2024-11-06 11:30:00+09:00 2024-11-06 12:30:00+09:00 2024-11-06 15:00:00+09:00
2024-11-07 2024-11-07 09:00:00+09:00 2024-11-07 11:30:00+09:00 2024-11-07 12:30:00+09:00 2024-11-07 15:00:00+09:00
```

### Additional Information
- **Previous Closing Time:** 15:00 PM (local time)
- **New Closing Time:** 15:30 PM (local time)
- References:
  - [JPX: Trading Rules of Domestic Stocks](https://www.jpx.co.jp/english/equities/trading/domestic/01.html#:~:text=Auction%20trading%20period%20is%20divided,05%20pm%20for%20each%20session.&text=Following%20days%20are%20holidays.)
  - [Notice of the transition](https://www.jpx.co.jp/english/equities/trading/domestic/tvdivq0000006blj-att/tradinghours_eg.pdf)

Thank you for your attention to this update!

**Comment by rsheftel** [comment]
Thank you for the update. If you want to submit a PR that would be great. If not I will definitely get to it when I have time.

**Comment by rsheftel** [comment]
Resolved by PR #351
